internal/pkg/mapper/template/replacekeys/mapper_test.go

```diff
@@ -0,0 +1,126 @@
+package replacekeys_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/mapper/template/replacekeys"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/state"
+	"github.com/keboola/keboola-as-code/internal/pkg/testdeps"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+func TestReplaceKeysMapper_OnRemoteChange(t *testing.T) {
+	t.Parallel()
+
```

Comment:
Pridany mapper, ktory nahradzuje objects keys pri (template) remote load.
Z testu by to malo byt jasne.